# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1411249062 14400
#      Sat Sep 20 17:37:42 2014 -0400
# Node ID 72e32208ff772dc150dc34844131dc6940bb9b47
# Parent  bc640d4678fb2d7afc3f96ba8c55c2269ea01123
Don't save build timestamp in doc-cache header comment (bug #43087)

* doc/interpreter/mk_doc_cache.m: Remove timestamp from header format
string.
* scripts/help/doc_cache_create.m: Define save_header_format_string
locally to be consistent with mk_doc_cache.m

diff --git a/doc/interpreter/mk_doc_cache.m b/doc/interpreter/mk_doc_cache.m
--- a/doc/interpreter/mk_doc_cache.m
+++ b/doc/interpreter/mk_doc_cache.m
@@ -128,10 +128,10 @@ for i = 2:n
   cache{1,k} = symbol;
   cache{2,k} = doc;
   cache{3,k} = first_sentence;
   k++;
 endfor
 
 cache(:,k:end) = [];    # delete unused pre-allocated entries
 
-save_header_format_string (["# doc-cache created by Octave " OCTAVE_VERSION ", %a %b %d %H:%M:%S %Y %Z"]);
+save_header_format_string (["# doc-cache created by Octave " OCTAVE_VERSION]);
 save ("-text", output_file, "cache");
diff --git a/scripts/help/doc_cache_create.m b/scripts/help/doc_cache_create.m
--- a/scripts/help/doc_cache_create.m
+++ b/scripts/help/doc_cache_create.m
@@ -50,16 +50,17 @@ function doc_cache_create (out_file = "d
   elseif (ischar (directory))
      cache = gen_doc_cache_in_dir (directory);
   else
      error ("doc_cache_create: second input argument must be a string or a cell of strings");
   endif
 
   ## Save cache
   if (! isempty (cache))
+     save_header_format_string (["# doc-cache created by Octave " OCTAVE_VERSION], "local");
      save ("-text", out_file, "cache");
   endif
 
 endfunction
 
 function [text, first_sentence, status] = handle_function (f, text, format)
   first_sentence = "";
   ## Skip functions that start with __ as these shouldn't be searched by lookfor
